sdk/core/test/integration/NetworkMember.test.ts

```diff
@@ -1094,4 +1094,49 @@ describe('CommonNetworkMember', () => {
     const commonNetworkMember = new AffinidiWallet(password, encryptedSeedElem, options)
     await commonNetworkMember.deleteAllCredentials()
   })
+
+  it('#createWallet should return wallet with resolvable did (elem)', async () => {
+    const commonNetworkMember = await AffinidiWallet.createWallet(
+      {
+        ...options,
+        didMethod: 'elem',
+      },
+      password,
+    )
+
+    const resolvedDidDocument = await commonNetworkMember.resolveDid(commonNetworkMember.did)
```

Comment:
Do I understand it right that this call would have failed in the current main branch?